client/src/components/Row.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import axios from '../services/axios';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import ImageContainer from './ImageContainer';
+
+function Row({ title, fetchUrl, isLarge }) {
+  const [movies, setMovies] = useState([]);
+
+  useEffect(() => {
+    async function fetchMovies() {
+      const request = await axios.get(fetchUrl);
+      setMovies(request.data.results);
+      return request;
+    }
```

Comment:
Is there a specific reason that you define the function within the useEffect hook? Usually we define the functions outside of the useEffect hook to allow reusability. 
If you need to call and define it from within you could also rewrite it to be an IIFE ([immediately invoked function expression](https://developer.mozilla.org/de/docs/Glossary/IIFE))

Reply:
You are right, the function won't be reused however so I figured the code looks cleaner that way :)